src/content/en/tools/chrome-devtools/javascript/step-code.md

```diff
@@ -174,14 +193,20 @@ Call stack with anonymous functions:
 
 ![Call stack with hard-to-read anonymous functions](imgs/anon.png)
 
-Call stack with named functions: 
+Call stack with named functions:
 
 ![Call stack with easier-to-read named function](imgs/named.png)
 
 <!-- blackbox OR disable third-party code??? -->
 
 ### Blackbox third-party code
 
+<aside class="warning">
+  <b>Warning:</b> This page is deprecated. See following section for up-to-date
+  information: <a href="reference#blackbox">Ignore a script or pattern of
+  scripts</a>
+</aside>
```

Comment:
I was using a custom callout in a another doc and then copy-pasted this over. No reason not to use `Warning: blah blah`